library/sqsdevCreds.properties

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
We can provide the same information from the properties.xml file for the application using the operator. This separate properties file is not needed.